STEERING-COMMITTEE.md

```diff
@@ -100,19 +97,25 @@ first name):
 \* Open seats have been allocated to the organization per the committee rules,
   but have not been assigned to a member of the organization.
 
-### Allocation of seats
+### Composition
+
+As of 2020, the three organizations with the largest demonstrated 
+commitment to the project are Google, IBM/RedHat, and VMWare. For
+2021, a 5 person steering committee should be competent to
+execute its responsibilities, with three seats for Google, one seat
+for IBM/RedHat, and one seat for VMWare.
 
-Seats on the steering committee are allocated based upon contribution to the
-project by an organization. No final decision has been made on the exact
-formula.
+### Nomination of representatives
 
-As the project continues to grow, we expect to add additional seats to the
-committee, ensuring that those contributing to the project are properly
-represented.
+Seats on the steering committee are allocated based upon lifetime
+contributions to the project by an organization. The intention is
```

Comment:
Lifetime contributions as a metric for representation feels like it is at odds with "being a 'truly open efffort'", because it makes it very difficult for a new contributing organization (or individual) to get a say in the policies that govern the organization. This method of measurement severely privileges the incumbent participants and *discourages* investment and engagement with the community.

I appreciate that it can feel unfair to have past contributions "age out" in terms of calculating representation, but this alternative seems overly conservative and against the interests of a healthy community.

If Google feels that there are *specific project issues* that need to be protected, it would be more productive to spell out those requirements and work on creating and empowering subcommittees to solve those problems and present them to Google for approval, rather than biasing the long-term governance in favor of older contributors.

Reply:
In particular, play through this _hypothetical_ thought experiment:

> VMware contributes for another 6 months and ends up with (lifetime) 40k contributions as measured by devstats. VMware then decides that they are going to make an opposite investment, and stop investing in Knative, including maintenance. How long would it take for a replacement participant to "earn their way" to that seat starting from zero (no WG leads, ec) if it were left empty by VMware? 2 years? 3 years?

Reply:
:-) this is partially why I was pushing for changes to the voting structure in Paul's PR

Reply:
Some type of dampening of “value” of contribution over time and even accelerating that dampening for contributors with lots of old contributions but less new ones could help. 

But then I imagine depending on dampening function, it will be a matter of playing around with the parameters to get to something fair and acceptable to all... I wonder if there is a precedence here in CNCF or other OSS organizations?

Reply:
The Kubernetes project looks for a certain number of contributions from the past year to determine how the community picks its steering body, eg: https://github.com/kubernetes/steering/blob/master/elections.md#eligibility-for-voting
One could imagine a variable number of minimum contributions required per year, with some non-zero and decreasing number of contributions in years N-2 and earlier allowing one a voice.